Components/alert.css

```diff
@@ -0,0 +1,50 @@
+.alert {
+    color: var(--white-color);
+    width: 100%;
```

Comment:
don't use color name itself in variable, give generic names. Else it kills the purpose of variables

Reply:
Thanks for the suggestion